### ISSUE TITLE
Snapshots not easily accessible

### ISSUE THREAD

**Issue by InI4** [osmdroid/osmdroid#1157]
## Issue Type

[X ] Question
[X ] Build system related

## Description and/or steps/code to reproduce the problem

1. BTW the https://github.com/osmdroid/osmdroid still talks about 6.0.0 being "the new release".

2. The real convenience problem. I have a build job which referes sonatype snapshot repository. I can find osmdroid snapshots on sonatype by browsing manually. In my gradle job I can easily edit the okhttp line  to switch between okhttp-3.11.0 and okhttp-3.12.0-SNAPSHOT with both builds working fine. However when I want to do the same switch with osmdroid i.e. switch between osmdroid-android:6.0.2 and osmdroid-android:6.0.3-SNAPSHOT, I only end up with gradle claiming osmdroid-android not found in the SNAPSHOT case.

To summarize, I have the right repository, cause I can see the snapshots there.
Switching between live and snapshot easily works with other libs.
Switching back to osmdroid snapshot does not work.

I have no clue, but a guess, some (gradle/maven) metadata is missing on osmdroid snapshot repository?

### Version of osmdroid the issue relates to:

SNAPSHOTs

**Comment by InI4** [comment]
Works:
`compile 'org.osmdroid:osmdroid-android:6.0.3-SNAPSHOT:debug@aar'`
